scripts/auth.ps1

```diff
@@ -0,0 +1,46 @@
+param (

```

Comment:
Please, add Set-StrictMode to your scripts. After refactoring, when you will provide PowerShell API with necessary functions, I believe that we can use it only once in main.ps1

Reply:
Also it would be nice to have #Requries statement for the PowerShell version, which you have used to create the script. Except version you can also force the user to use administrator rights if needed and do other cool things.

Reply:
D O N E